File-Structure.md

```diff
@@ -0,0 +1,25 @@
+Here is the recommended Folder/File structure in order to ensure your roms are properly identified.  Failure to follow one of these formats **may** work, but is not guaranteed nor supported.
+
+*library* refers to RomM's internal container path, just replace it in your example with the directory containing your roms.
+
+#### Recommended Format
+
+This is the ideal structure to follow, every new installation should follow this unless absolutely necessary.
```

Comment:
It can be really finicky about whether or not it will detect the roms if not in one of the supported structures, unless you modify the config,yml.   For this reason, I used this wording to highly discourage custom structures that would cause support headaches.  Maybe arcaneasada might have some input. and can correct this.